src/components/ChatLog.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+// const messageIndex = 0;
+
+const ChatLog = (props, onLikeCallback) => {
+  const chatComponents = props.entries.map((chat, index) => {
+    return (
+      <ChatEntry
+        key={index}
```

Comment:
If our messages have unique `id`s, another option would be to use the message `id` as the `key` rather than an external `index`.